index.rst

```diff
@@ -1,61 +1,365 @@
+
+IDM-2007
+    Owners must be able to delete groups.
+
+IDM-2008
+    Users must be able to see all of their group memberships and their expirations (if any).
+
+Quotas
+------
+
+IDM-2100
+    Groups may have one or more quota grants associated with the group.
+    These are of two types: Quotas for the group itself (such as for shared storage space), and quota that is inherited by every member of the group (granting additional personal quota).
+    These may represent file storage quotas or any other service limit that may vary by user (API rate limits, CPU equivalents for batch jobs, download size limits, or whatever may eventually be appropriate).
+    The identity management system need not understand the quotas, but it should be able to sum multiple quotas under the same label.
+
+IDM-2101
+    All members of the group must be able to view all of its quota grants.
+
+IDM-2102
+    All owners of the group must be able to request new quota grants.
+    That request should be routed to some approval process by a manager of the relevant resource, who can then grant or deny the request via the identity management web interface.
+
+IDM-2103
+    Quota grants may expire.
+    The owners of a group must be notified via email of pending quota grant expirations.
+
+Logging
+-------
+
+IDM-2200
+    All group creation, deletion, renaming, and membership changes must be logged.
+
+IDM-2201
+    Owners must be able to see, via the web interface, all history of changes to the group.
+
+IDM-2202
+    All changes to quotas associated with groups must be logged.
+
+IDM-2203
+    Group owners must be able to see a history of all changes to their group quotas.
+
+IDM-2204
+    Users must be able to see a history of all changes to their group membership.
 
-.. TODO: Delete the note below before merging new content to the master branch.
+Administration
+--------------
 
-.. note::
+IDM-2300
+    Administrators must be able to create groups, delete groups, rename groups, and change the membership of any group.
 
-   **This technote is not yet published.**
+IDM-2301
+    Administrators must be able to change the quota grants and requests for any group.
 
-   Requirements for the identity management component of the Science Platform.
+IDM-2302
+    Administrators must be able to impersonate a user and see exactly the same group management and display screens that the user would see.
 
-.. Add content here.
-.. Do not include the document title (it's automatically added from metadata.yaml).
+API
+===
 
-.. .. rubric:: References
+IDM-3000
+    The identity management system must provide a read-only API to other Science Platform components.
+    That API, when given a user authenticator (a token or cookie), must return the user metadata, group memberships, and individual and group quota information.
```

Comment:
If we want to implement [IVOA GMS](http://ivoa.net/documents/GMS/index.html) we can do this as a layer on top of this API.

(This is just an observation - I'm not suggesting that you have to mention it here.)